docs/README.md

```diff
@@ -97,6 +97,16 @@ To help you convert HTML snippets to Elm code and help newcomers learn the synta
 
 ---
 
+### __Go to symbol__
+
+__Setting:__ `elmLand.feature.goToSymbol`
+
+You can navigate symbols inside a file. This is helpful for quickly navigating among functions, values and types in a file.
+
+![Go to symbol](./go-to-symbol.gif)
```

Comment:
TODO: Create this gif

Reply:
TODO: Update the performance tables further below

(should probably be done on the same computer as the other features?)